### PR TITLE
added to js

### PR DESCRIPTION
**Pull request by KarinaMireles**

### CODE REVIEW

File: script.js/script.js [hunk 1]
```diff
@@ -18,4 +20,37 @@ const menuItems = [
...
+  const item = document.createElement("div");
+  item.className = "menu-item";
+  item.textContent = `${product.name} - $${product.price.toFixed(2)}`;
+  item.addEventListener("click", () => addToCart(product));
+  menu.appendChild(item);
+});
+
 const cartItems = [];
+
+function addToCart(product) {
```
Comment (Jpropst) [inline script.js/script.js]: //could we do this instead?
const addToCart = (product) => {

File: script.js/script.js [hunk 2]
```diff
@@ -18,4 +20,37 @@ const menuItems = [
...
+  cartItems.push(product);
+  updateCartDisplay();
+}
+function updateCartDisplay() {
```
Comment (Jpropst) [inline script.js/script.js]: same here?

File: script.js/script.js [hunk 3]
```diff
@@ -18,4 +20,37 @@ const menuItems = [
...
+  cart.innerHTML = "";
+  cartItems.forEach((product) => {
+    const cartItem = document.createElement("div");
+    cartItem.className = "cart-item";
+    cartItem.textContent = product.name;
+    cart.appendChild(cartItem);
+  });
+}
+checkoutButton.addEventListener("click", () => {
```
Comment (Jpropst) [inline script.js/script.js:46]: not sure if it matters but can we declare this with const as well?

File: script.js/script.js [hunk 4]
```diff
@@ -18,4 +20,37 @@ const menuItems = [
...
+  updateCheckoutSummary();
+  checkoutModal.style.display = "block";
+});
+processPaymentButton.addEventListener("click", () => {
```
Comment (Jpropst) [inline script.js/script.js:50]: and here?

File: script.js/script.js [hunk 5]
```diff
@@ -18,4 +20,37 @@ const menuItems = [
...
+  const paymentMethod = paymentMethodSelect.value;
+  processPayment(paymentMethod);
+});
+modalClose.addEventListener("click", () => {
```
Comment (Jpropst) [inline script.js/script.js:54]: and here?

**Review by Jpropst — CHANGES_REQUESTED** [review]
Peep my comments

File: script.js/script.js (hunk 5 above)
Comment (KarinaMireles) [reply script.js/script.js:54]: fixed styling

**Review by Jpropst — APPROVED** [review]
LGTM :)

**Pull request merged**